cpp/include/cudf/strings/extract.hpp

```diff
@@ -27,14 +27,14 @@ namespace strings {
 struct regex_program;
 
 /**
- * @addtogroup strings_substring
+ * @addtogroup strings_extract
  * @{
  * @file
  */
 
 /**
  * @brief Returns a table of strings columns where each column corresponds to the matching
- * group specified in the given regular expression pattern.
+ * group specified in the given regex_program object
```

Comment:
```suggestion
 * group specified in the given regex_program object.
```

Reply:
These are not sentences and so do not warrant a period.